mitodl/micromasters
# Updated realistic user and program data to add more fake users

#### What are the relevant tickets?

No ticket. Done in response to #2336 
Rebased on #2316 

#### What's this PR do?

Ups the total fake user count to 120, and makes sure that each of the fake programs has over 50 user enrolled. This will ensure that the pagination control should show up.

#### How should this be manually tested?

Unseed, re-seed, reindex, and check /learners. Make sure both fake programs have page-able results
